src/components/MenuEnvelope.vue

```diff
@@ -562,6 +571,12 @@ export default {
 		setCustomSnooze() {
 			this.onSnooze(this.customSnoozeDateTime.valueOf())
 		},
+		onPrint() {
+			// needed for the actions menu to actually close and not be shown in the print preview
+			setTimeout(() => {
+				window.print()
+			}, 10)
```

Comment:
Could you use Vue's $nextTick callback?

Reply:
Doing
```js
this.$nextTick(() => {
	window.print()
})
```
Does not seem to work, is there a better way to do it?

Reply:
The only thing I could imagine is adding a dedicated close-after-click event in the upstream component and listening to that instead of the click event